cinema/views.py

```diff
@@ -1,108 +1,71 @@
-                         f"{person.user__last_name}"}
-            )
-film_fields = (
-    "country__name", "genre__name", "language__name",
-    "distributor__name", "news__pk", "news__title"
+from .models import (
+    Product,
+    News,
+    Film,
+    CinemaPerson,
+    MpaaRating
+)
+from .services import (
+    get_cast_and_crew,
+    get_films_info,
+    get_persons_info,
+    get_films_ratings_sets,
+    get_celebrity_news_id, get_filmography_and_extra_info
 )
-info_by_films_qs = all_films.values_list(
-    "pk", *film_fields, named=True
-).order_by("-news__created_at")
-
-info_by_films = {}
-for film in info_by_films_qs:
-    if film.pk not in info_by_films:
-        info_by_films[film.pk] = {}
-
-    for field in film_fields:
-        if field not in ("news__pk", "news__title"):
-            if field not in info_by_films[film.pk]:
-                info_by_films[film.pk][field] = []
-            if field == "country__name" and film.country__name not in \
-                    info_by_films[film.pk][field]:
-                info_by_films[film.pk][field].append(film.country__name)
-            elif field == "genre__name" and film.genre__name not in \
-                    info_by_films[film.pk][field]:
-                info_by_films[film.pk][field].append(film.genre__name)
-            elif field == "language__name" and film.language__name not in \
-                    info_by_films[film.pk][field]:
-                info_by_films[film.pk][field].append(film.language__name)
-            elif field == "distributor__name" and film.distributor__name \
-                    not in info_by_films[film.pk][field]:
-                info_by_films[film.pk][field].append(film.distributor__name)
-        else:
-            if "news" not in info_by_films[film.pk]:
-                info_by_films[film.pk]["news"] = {}
-            if field == "news__pk" and film.news__pk and film.news__pk not in \
-                    info_by_films[film.pk]["news"]:
-                info_by_films[film.pk]["news"][film.news__pk] = film.news__title
 
-all_persons = CinemaPerson.objects.prefetch_related(
-    "film_set__genre", "film_set__imdb_rating", "news_set"
-).select_related("user")
+DESCRIPTION_BLU_RAY = "Welcome to the fantastic Blu-ray department here, " \
+                      "where you can find the films, you could ever want to " \
+                      "see in superb high definition quality. We're " \
+                      "constantly adding all of the latest Blu-ray releases " \
+                      "to our collection and always have an unbelievable " \
+                      "offer or two. We promise great deals on the best films!"
+DESCRIPTION_NEWS = "The latest movie news on the movies 'you're most " \
+                   "interested in seeing."
+
+films_cast_and_crew = get_cast_and_crew()
+films_info = get_films_info()
+persons_info = get_persons_info()
+films_ratings_sets = get_films_ratings_sets()
+celebrity_news_id = get_celebrity_news_id()
```

Comment:
Leave it for now, but honestly, I did not like it absolutely.

Reply:
ok